src/App.scss

```diff
@@ -33,7 +33,10 @@
     border: #444444;
 }
 .bg-photo {
+    min-height: 200vh;
```

Comment:
why 2x the viewport height?

Reply:
I wanted to have same background across components so if you scroll bg does not change